models/conv_dropout.py

```diff
@@ -0,0 +1,64 @@
+import random
+import collections
+import warnings
+from six.moves import range
+import numpy as np
+import six
+import tensorflow as tf
+import tensorflow_federated as tff
+from models.model import Model
+
+
+class ConvDropoutSupervisedModel(Model):
```

Comment:
Why not just ConvSupervisedModel? you can add the dropout ratio to config. When the dropout ratio is 0.0 or 1.0 (i'm not sure which) its just a regular convolution model.

Reply:
That can be done, but this is easier for experimentation purposes for now. Especially if there are multiple dropout layers with different dropout ratios.